### PR TITLE
test: extend external quote source

### PR DESCRIPTION
**Pull request by 0xged**

### CODE REVIEW

File: test/integration/services/quotes/sources/quote-sources.spec.ts [hunk 1]
```diff
@@ -104,6 +104,18 @@ describe.skip('Quote Sources [External Quotes]', () => {
...
       describe('Sell order', () => {
+        quoteTest({
+          test: Test.SELL_RANDOM_ERC20_TO_STABLE,
+          when: 'swapping 100 random token to stable',
+          request: {
+            sellToken: RANDOM_ERC20,
+            buyToken: nativeToken,
+            order: {
+              type: 'sell',
+              sellAmount: parseUnits('100', 18),
```
Comment (0xsambugs) [inline test/integration/services/quotes/sources/quote-sources.spec.ts:115]: What would you say if we change this to `unitsToSell: number`, and then, inside the test, we can do `parseUnits('${unitsToSell}', sellToken.decimals)`
So we don't have to worry about decimals here

**Review by 0xsambugs — COMMENTED** [review]
Small suggestion to support different types of tokens

File: test/integration/services/quotes/sources/quote-sources.spec.ts (hunk 1 above)
Comment (0xged) [reply test/integration/services/quotes/sources/quote-sources.spec.ts:115]: That was my initial thought, but I suck at typing, so I couldn't achieve it 😬

Comment (0xsambugs) [reply test/integration/services/quotes/sources/quote-sources.spec.ts:115]: I'll do it

**Comment by 0xsambugs** [comment]
That should do it @0xged . If you are ok with the changes, feel free to merge 👍

**Pull request merged**